python/dgl/data/lrgb.py

```diff
@@ -450,3 +457,234 @@ def __getitem__(self, idx):
             return self.graphs[idx], self.labels[idx]
         else:
             return self._transform(self.graphs[idx]), self.labels[idx]
+
+
+class VOCSuperpixelsDataset(DGLDataset):
+    r"""VOCSuperpixels dataset for the node classification task.
+
+    DGL dataset of Pascal VOC Superpixels which contains image superpixels
+    and a semantic segmentation label for each node superpixel.
+
+    color map
+    0=background, 1=aeroplane, 2=bicycle, 3=bird, 4=boat, 5=bottle,
+    6=bus, 7=car, 8=cat, 9=chair, 10=cow,
+    11=diningtable, 12=dog, 13=horse, 14=motorbike, 15=person,
+    16=potted plant, 17=sheep, 18=sofa, 19=train, 20=tv/monitor
+
+    Reference `<https://arxiv.org/abs/2206.08164.pdf>`_
+
+    Statistics:
+
+    - Train examples: 8,498
+    - Valid examples: 1,428
+    - Test examples: 1,429
+    - Average number of nodes: 479.40
+    - Average number of edges: 2,710.48
+
+    Parameters
+    ----------
+    raw_dir : str
+        Raw file directory to download/contains the input data directory.
+        Default: "~/.dgl/".
+    split : str
+        Should be chosen from ["train", "val", "test"]
+        Default: "train".
+    construct_format : str, optional
+        Option to select the graph construction format.
+        Should be chosen from ["edge_wt_only_coord", "edge_wt_coord_feat", "edge_wt_region_boundary"]
+        "edge_wt_only_coord": the graphs are 8-nn graphs with the edge weights
+        computed based on only spatial coordinates of superpixel nodes.
+        "edge_wt_coord_feat": the graphs are 8-nn graphs with the edge weights
+        computed based on combination of spatial coordinates and feature
+        values of superpixel nodes.
+        "edge_wt_region_boundary": the graphs region boundary graphs where two
+        regions (i.e. superpixel nodes) have an edge between them if they share
+        a boundary in the original image.
+        Default: "edge_wt_region_boundary".
+    slic_compactness : int, optional
```

Comment:
```suggestion
    slice_compactness : int, optional
```

Reply:
the slic corresponds to the SLIC algorithm